kernel/aster-nix/src/vm/vmo/mod.rs

```diff
@@ -311,6 +322,7 @@ impl Vmo_ {
         range: &Range<usize>,
         mut operate: F,
         will_write: bool,
+        no_commit: bool,
     ) -> Result<()>
```

Comment:
After the current modifications, there are too many boolean variables in the parameters, which can easily affect the readability at the call site of this API. I suggest using a new abstraction like`CommitFlags`, which includes properties like `will_write`, `no_commit`, etc., and using it as a parameter.

Reply:
In the latest commit, I use `bitflags` to describe the parameter `CommitFlags`.